gatsby-template/gatsby-config.js

```diff
@@ -12,8 +12,8 @@ const configuration = {
   bio: `{{ bio }}`,
   author: '{{ authorName }}',
   githubUrl: '{{ githubURL }}',
-  // replace this by the url where your website will be published
-  siteUrl: 'http://localhost:8000',
+  // fallback option if NetlifyURL isnt provided
```

Comment:
The comment should describe what the field is about

```suggestion
  // the URL where your website is published
  // it is used to show an "Discuss on twitter" link
```